engines/director/channel.cpp

```diff
@@ -291,16 +291,6 @@ void Channel::setClean(Sprite *nextSprite, int spriteId, bool partial) {
 			}
 		}
 
-		// If the previous sprite in this channel was a video, but the new sprite is different,
-		// stop the playback.
-		// TODO: In D4 *any* channels playing the same cast member across a frame change seems
```

Comment:
I wonder, what kind of side effects it could bring, since now we're not stopping the video playback.

Could you please elaborate a bit?

Reply:
I'll elaborate. The logic used to check if a video should be stopped is broken for Majestics case. 
The video is stopped before it can be played. 

What I've seen so far is that a video stops playing when the movieRate is 0. 
Either because the movie was finished, or the movieRate as set to 0.

It seems to me the .stop() in [1] replaced the functionality meant here.
Which calls [2] directly, instead of indirectly. 

Do you know of any Director files that rely on the original behavior that I could check against?

[1]https://github.com/scummvm/scummvm/blob/master/engines/director/castmember.cpp#L421
[2]https://github.com/scummvm/scummvm/blob/master/video/video_decoder.cpp#L422